gen_defaults/cbl-defaults.json

```diff
@@ -82,6 +92,81 @@
                     "subset": "enum"
                 },
                 "description": "By default, vectors are compared using Euclidean metrics"
+            },
+            {
+                "name": "MinTrainingSize",
+                "value": 0,
+                "type": {
+                    "id": "uint",
+                    "subset": "system"
+                },
+                "type_swift": {
+                    "id": "UInt32",
+                    "subset": "system"
+                },
+                "type_c": {
+                    "id": "unsigned",
+                    "subset": "system"
+                },
+                "type_objc": {
+                    "id": "unsigned int",
+                    "subset": "system"
+                },
+                "type_csharp": {
+                    "id": "int",
```

Comment:
This is incorrect.  C# should be uint, so this (and C) should be removed as the default generated type is fine.